content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -0,0 +1,163 @@
+   :alt: Creating an SMS marketing template.
+
+First, give the mailing a :guilabel:`Subject`, which describes what the mailing is about.
+
+Next, in the :guilabel:`Recipients` field, choose to whom this SMS will be sent. By default, Odoo
+has :guilabel:`Mailing List` selected. If this is the desired :guilabel:`Recipients` field option,
+specify which mailing list Odoo should send this SMS to, in the :guilabel:`Select Mailing List`
+field to the right.
+
+.. note::
+   To create (or edit) a mailing list, go to :menuselection:`Mailing Lists --> Mailing List`. There,
+   Odoo displays all previously created mailing lists, along with various types of data related to
+   that specific list (e.g. number of contacts, mailings, recipients, etc.).
+
+.. image:: sms_essentials/sms-mailing-list.png
+   :align: center
+   :alt: View of the mailing list page in the SMS marketing application.
+
+To learn more about mailing lists (and contacts) check out :doc:`mailing_lists_blacklists`.
+
+To reveal all the possible options in the :guilabel:`Recipients` field, click the field to see all
+the choices Odoo makes avaialble. When another field (other than :guilabel:`Mailing List`) is
+selected, the option to specify that chosen field even further becomes available, via an
+:guilabel:`Add Filter` button.
+
+Clicking the :guilabel:`Add Filter` button, reveals fully customizable domain rule fields, which can
+be configured similar to an equation. You can create multiple recipient rules, if necessary. Then,
+Odoo will be sure to only send the SMS to recipients who fit into whatever criteria is configured in
+those fields.
+
+For instance, if :guilabel:`Contact` is chosen, all of the Odoo contacts in the database (vendors,
+customers, etc.) will receive the SMS, unless a specific recipient rule(s) are entered.
+
+In the example below, the message will only be sent to contacts who haven't chosen to
+:guilabel:`Blacklist` their contact information through the :guilabel:`Unsubscribing` process.
+
+.. image:: sms_essentials/contact-recipient.png
+   :align: center
+   :alt: Contact recipients on SMS marketing.
+
+How to create SMS messages
+==========================
+
+Enter the content of the SMS in the text field, found in the :guilabel:`SMS Content` tab. Links and
+emojis can also be included. Beneath the text field, Odoo displays how many characters are used in
+the message, along with how many SMS mailings it will take to deliver the complete message.
+
+.. tip::
+   To check the price of sending an SMS for a country, click on the :guilabel:`Information` icon.
+
+.. image:: sms_essentials/sms-price-check.png
+   :align: center
+   :alt: SMS price check icon
+
+.. note::
+   Credits must be purchased from Odoo in order to take advantage of the app (and its features).
+   SMS messages will not be sent without credits.
+
+.. seealso::
+   - :doc:`../pricing/pricing_and_faq`
+
+How to track links used in SMS messages
+---------------------------------------
```

Comment:
Note for all of the headings that start with `How to...`
We can just remove that part and stick with the action that the heading conveys for the section. Please apply to all headings in this and other docs.

```suggestion
Track links used in SMS messages
--------------------------------
```